pkg/apis/experimental/v1alpha1/custom_builder_types.go

```diff
@@ -21,10 +22,10 @@ type CustomBuilder struct {
 }
 
 type CustomBuilderSpec struct {
-	Tag   string  `json:"tag"`
-	Stack Stack   `json:"stack"`
-	Store Store   `json:"store"`
-	Order []Group `json:"order"`
+	Tag   string                 `json:"tag"`
+	Stack Stack                  `json:"stack"`
+	Store corev1.ObjectReference `json:"store"`
```

Comment:
Because, these are just cluster scoped should we just provide a store name here?